packages/shared/src/components/filters/TagsFilter.tsx

```diff
@@ -43,13 +60,16 @@ export default function TagsFilter({
     };
   }, [feedSettings]);
 
-  const tagUnblockAction = ({ tags }) => {
-    setUnblockItem({
-      tag: tags,
-      action: () => onUnblockTags({ tags }),
+  const tagUnblockAction = ({ tags: [tag] }: TagActionArguments) => {
+    if (shouldShowMyFeed && !user) {
+      return onUnblockTags({ tags: [tag] });
+    }
```

Comment:
Why is this a different action based on the shouldShowMyFeed?

Reply:
Since blocking as an anonymous user doesn't take effect on the actual feed, I think it is not necessary to ask anymore, especially the question says the source will now be seen in the feed which is still seen even after blocking.

Reply:
Ok, but what does it have to do with the My Feed?

Reply:
When unblocking a tag they have blocked.